src/dfcx_scrapi/tools/utterance_generator_util.py

```diff
@@ -18,8 +18,8 @@
 import string
 import pandas as pd
 from typing import List
-from dfcx_scrapi.core.scrapi_base import ScrapiBase
-from dfcx_scrapi.core.intents import Intents
+from dfcx_scrapi.core import scrapi_base
+from dfcx_scrapi.core import intents
 from dfcx_scrapi.core_ml.utterance_generator import UtteranceGenerator
```

Comment:
@kmaphoenix  should be:
`from dfcx_scrapi.core_ml import utterance_generator `

Reply:
ack, updated.